Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/TestImportApiHelper.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using kCura.Relativity.Client;
+using kCura.Relativity.Client.DTOs;
+using NUnit.Framework;
+using Relativity.API;
+using Relativity.Test.Helpers.SharedTestHelpers;
+
+namespace Relativity.Test.Helpers.NUnit.Integration
+{
+	public class TestImportApiHelper
+	{
+		private TestHelper Sut;
+
+		[OneTimeSetUp]
+		public void Setup()
+		{
+			Sut = new TestHelper(ConfigurationHelper.ADMIN_USERNAME, ConfigurationHelper.DEFAULT_PASSWORD);
+		}
+
+		[OneTimeTearDown]
+		public void Teardown()
+		{
+			Sut = null;
+		}
+
+		[Test]
+		public void CreateDocumentsWithFolderName()
+		{
+			// Arrange
+			int workspaceId = ConfigurationHelper.WORKSPACEID;
```

Comment:
this workspace ID should come from a new workspace you create using helpers and not from the app config file